### DISCUSSION TITLE
Paint asphalt

### DISCUSSION THREAD

**Discussion by javi-r-p** [MisterJulsen/TrafficCraft#2] [Q&A]
Hi! I'm playing minecraft 1.18.2 with this mod and I have an issue, I'm trying to paint lines and arrows in the asphalt but I can only paint the asphalt with a single colour and I don't know how to paint lines and figures. Could someone help me? Thanks!

![2023-12-15_13 40 54](https://github.com/MisterJulsen/TrafficCraft/assets/117043352/c6a239e2-d2d4-4224-bfc0-316735de161a)

**Comment by MisterJulsen** [comment]
Hi, if you right-click in the air with your brush while holding down the shift key, a menu should open where you can select a pattern.
